Lib/test/test_funcattrs.py

```diff
@@ -93,6 +93,27 @@ def f(): print(a)
             self.fail("shouldn't be able to read an empty cell")
         a = 12
 
+    def test_set_cell(self):
+        a = 12
+        def f(): return a
+        c = f.__closure__
+        c[0].cell_contents = 9
+        self.assertEqual(c[0].cell_contents, 9)
+        self.assertEqual(f(), 9)
+        del c[0].cell_contents
+        try:
```

Comment:
I understand that you have copied an example from the above test, but this is too verbose. `with self.assertRaises(ValueError):` is much more compact.